src/cloud/docker/docker-config.md

```diff
@@ -103,14 +103,8 @@ echo "127.0.0.1 magento2.test" | sudo tee -a /etc/hosts
 
 ## Set up email
 
-Send emails from your Docker environment by adding the following configuration to the `docker-compose.yml` configuration file:
-
-```yaml
-ENABLE_SENDMAIL=true
-```
-
-{:.bs-callout-warning}
-We do not recommend using Sendmail on CLI containers because the service can slow down the container creation process.
+The [MailHog] service included by default as replacement for Sendmail. 
```

Comment:
```suggestion
The default {{ site.data.var.mcd-prod }} configuration includes the [MailHog] service as a replacement for the Sendmail service because Sendmail can cause performance issues in the local  Docker environment.
```